datacenterlight/views.py

```diff
@@ -444,22 +444,37 @@ def post(self, request, *args, **kwargs):
         billing_address_data = request.session.get('billing_address_data')
         billing_address_id = request.session.get('billing_address')
         vm_template_id = template.get('id', 1)
-        final_price = specs.get('price')
 
         # Make stripe charge to a customer
         stripe_utils = StripeUtils()
-        charge_response = stripe_utils.make_charge(amount=final_price,
-                                                   customer=customer.stripe_id)
-
-        # Check if the payment was approved
-        if not charge_response.get('response_object') and not charge_response.get('paid'):
-            msg = charge_response.get('error')
-            messages.add_message(self.request, messages.ERROR, msg, extra_tags='make_charge_error')
-            return HttpResponseRedirect(reverse('datacenterlight:payment') + '#payment_error')
-
-        charge = charge_response.get('response_object')
+        cpu = specs.get('cpu')
+        memory = specs.get('memory')
+        disk_size = specs.get('disk_size')
+        amount_to_be_charged = (cpu * 5) + (memory * 2) + (disk_size * 0.6)
```

Comment:
I wonder if it is a good idea to store the calculation here - do we have a more central place for it? stripe_utils might not be entirely fitting, but something in that direction.

The motivation is: maybe we want to have a multiple versions online or at least next to each other / maybe we want have multiple active version.

Maybe my critic is wrong in this place and we should just create a new ticket for this and fix it another time ;-)